interview-questions.md

```diff
@@ -8,44 +8,46 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a class?
 
-Your answer:
+Your answer: Calls an attribute from the initialized class
```

Comment:
An expanded answer would show your interviewer the depth of your understanding.  One-liner answers will often not impress interviewers.  Consider talking about a time you have used super and/or providing examples.